test/cli/test_dump_fields.rb

```diff
@@ -0,0 +1,206 @@
+# frozen_string_literal: true
+
+require_relative "test_helper"
```

Comment:
Change this to `require "helpers"` and remove the `require "aruba"` here and below, and this PR should be good to go.